spec/tests/ContentMetadata.spec.ts

```diff
@@ -1,5 +1,5 @@
 /// <reference path='../../src/ts/Conviva.d.ts'/>
-import { PlayerType } from 'bitmovin-player';
+import {PlayerType, StreamType, VRContentType} from 'bitmovin-player';
```

Comment:
Please stick to the existing code style 🙂 
```suggestion
import { PlayerType, StreamType, VRContentType } from 'bitmovin-player';
```

Reply:
Done